### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_Screenreader:Voice over is announcing incorrect data when we operate with spinner.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6030]
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
Tool Name: Voice Over
MAS Reference: MAS1.3.1
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the SQL agent tab from Home page->Click on new job button->Select steps tab.
4.Go to the "New step" button hit enter/spacebar
5.Navigate to "Advanced' button and navigate with Tab till spinner.
5.Start Voice over
6.Observed that Voice over is not announcement of data while accessing spinner.
**Actual:**
Voice over is announcing incorrect data when we operate with Soinner.it is not announcing data as per selected by spinner. If we select 9 then it is showing 9 10.

**Similar issue has been observed for below page** -
**1. CreateAlertOptions -** 
Go to   the  SQL server agent tab->On alerts   tab->Click on new alert button->Select Options tab.

**Expected:**
Voice Over should announce the data as per selection like when we select 9 it should announce 9.
**User Impact:**
User with disabilities will be confuse with the dual information announced by the voice over.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
**MAS1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6030_Accessibility]A11y_AzureDataStudio_NewJobSteps_Screenreader.pptx](https://github.com/microsoft/azuredatastudio/files/3290647/6030_Accessibility.A11y_AzureDataStudio_NewJobSteps_Screenreader.pptx)

Does this issue occur when all extensions are disabled?: Yes
